contrib/design-docs/podman6hld.md

```diff
@@ -0,0 +1,247 @@
+# Podman 6 High Level Design
+
+#
+
+The following document describes the high level design of Podman 6\. Podman 6 will be generally available in Spring of 2026\.  In this document, I try to summarize:
+
+* Breaking changes in API or CLI for Podman
+* New functionality that is non-trivial work.
+* Impacts to other components (like netavark, common)
+* Deprecations
+
+The content in this document are proposals generally.  If a proposal is objectionable to a considerable number of people or stakeholders, then it will be reconsidered and decisions will be made consensually.  Also, new content certainly may be added during review.  Moreover, some of the features described below may have their own design documents where more detail will be provided.
+
+[**Branching, Schedules, Podman 5**](#Branching-Schedules-Podman-5)
+
+[**Deprecations**](#deprecations)
+
+[**Configuration files**](#configuration-files)
+
+[**Changes in Defaults for Podman**](#changes-in-defaults-for-podman)
+
+[**Libkrun**](#libkrun)
+
+[**New Conmon**](#new-conmon)
+
+[**Pasta**](#pasta)
+
+[**Machine**](#machine)
+
+[**Netavark**](#netavark)
+
+[**Podman machine updates**](#podman-machine-updates)
+
+[**Issues in containers/podman with "6.0" Label**](#issues-in-containers/podman-with-"6.0"-label)
+
+
+# Branching, Schedules, Podman 5
+
+We have spent considerable time discussing when the Podman main branch should be open for Podman 6\.  To understand this, we must understand the schedule for the upcoming months.  We still have active development occurring for Podman 5 series releases.  We have the following Podman 5 minor released scheduled:
+
+* Podman 5.7 \- November 2025 (first week)
+* Podman 5.8 \- February 2026
+
+With Podman 5.7 releasing in early November, we will create a 5.7 branch when releasing the first release candidate.  Current plans project RC1 on the week of October 20th.  Given a short period for things to settle down, we can then branch Podman main for 6.0 development the week of October 27th.
+
+Podman 5.8 will focus on critical bug fixes and user requirements. As such, it will be derived from the Podman 5.7 branch in February.
+
+Podman 6 is to become generally available in the Spring of 2026\.  The exact release date is still to be determined, however, the intent is to release it in Fedora 44 so likely completion date is in the March 2026 timeframe depending on the allowances Fedora permits us.
+
+# Deprecations
+
+The following previously announced deprecations will be made final.  Some have already been communicated to users, whereas some will be new to Podman users.
+
+## **Slirp4netns**
+
+[Slipr4netns](https://github.com/rootless-containers/slirp4netns) was the original network utility for rootless Podman and has been replaced by [Pasta](https://passt.top/passt/about/).  Podman has been able to use Pasta since version 4.4.  In Podman 5.0, Pasta became the default rootless option and a deprecation notice for slirp4netns was made in Podman 5\.
```

Comment:
_a deprecation notice for slirp4netns was made in Podman 5_

The text looks good but I'm curious where to find the deprecation notice. I searched in https://github.com/containers/podman/blob/main/RELEASE_NOTES.md
but couldn't find it. I also did some recursive grep


```
grep -ri slirp4netns docs/
```

```
grep -ri deprecat docs/
```

Maybe I just missed it?